modules/igv/1.0/igv.smk

```diff
@@ -0,0 +1,824 @@
+)
+
+# Rename genome_build values in sample metadata to correlate with MAF values
+CFG["runs"]["tumour_genome_build"].mask(CFG["runs"]["tumour_genome_build"].isin(CFG["options"]["genome_map"]["grch37"]), "grch37", inplace=True)
+CFG["runs"]["tumour_genome_build"].mask(CFG["runs"]["tumour_genome_build"].isin(CFG["options"]["genome_map"]["hg38"]), "hg38", inplace=True)
+CFG["runs"]["normal_genome_build"].mask(CFG["runs"]["normal_genome_build"].isin(CFG["options"]["genome_map"]["grch37"]), "grch37", inplace=True)
+CFG["runs"]["normal_genome_build"].mask(CFG["runs"]["normal_genome_build"].isin(CFG["options"]["genome_map"]["hg38"]), "hg38", inplace=True)
+
+# Setup variables 
+SUFFIX = ".pad" + str(CFG["options"]["generate_batch_script"]["padding"])
+if "launch_date" in CFG:
+    LAUNCH_DATE = CFG["launch_date"]
+else:
+    LAUNCH_DATE = datetime.today().strftime('%Y-%m-%d')
+
+# Define rules to be run locally when using a compute cluster
+localrules:
+    _igv_symlink_bams,
+    _igv_symlink_maf,
+    _igv_reduce_maf_cols,
+    _igv_merge_regions,
+    _igv_format_regions,
+    _igv_liftover_regions,
+    _igv_merge_lifted_regions,
+    _igv_filter_maf,
+    _igv_create_batch_script_per_variant,
+    _igv_batches_to_merge,
+    _igv_download_igv,
+    _igv_run,
+    _igv_track_failed,
+    _igv_quality_control,
+    _igv_symlink_snapshot,
+    _igv_check_snapshots,
+    _igv_check_samples,
+    _igv_setup_estimates,
+    _igv_estimate_snapshots,
+    _igv_check_estimates
+
+
+##### FUNCTIONS #####
+
+
+def get_bam(wildcards):
+    metadata = config["lcr-modules"]["igv"]["samples"]
+    return expand("data/{{seq_type}}_bams/{{sample_id}}.{genome_build}.bam", genome_build=metadata[(metadata.sample_id == wildcards.sample_id) & (metadata.seq_type == wildcards.seq_type)]["genome_build"])
+
+def get_bai(wildcards):
+    metadata = config["lcr-modules"]["igv"]["samples"]
+    return expand("data/{{seq_type}}_bams/{{sample_id}}.{genome_build}.bam.bai", genome_build=metadata[(metadata.sample_id == wildcards.sample_id) & (metadata.seq_type == wildcards.seq_type)]["genome_build"])
+
+def get_maf(wildcards):
+    unix_group = config["unix_group"]
+    return expand(config["lcr-modules"]["igv"]["inputs"]["maf"], allow_missing=True, unix_group=unix_group)
+
+
+##### RULES #####
+
+
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _igv_symlink_bams:
+    input:
+        bam = get_bam,
+        bai = get_bai
+    output:
+        bam = CFG["dirs"]["inputs"] + "bams/{seq_type}/{sample_id}.bam",
+        bai = CFG["dirs"]["inputs"] + "bams/{seq_type}/{sample_id}.bam.bai"
+    run:
+        op.absolute_symlink(input.bam, output.bam)
+        op.absolute_symlink(input.bai, output.bai)
+
+rule _igv_symlink_maf:
+    input:
+        maf = get_maf
+    output:
+        maf = CFG["dirs"]["inputs"] + "maf/{seq_type}--{genome_build}/{tumour_id}--{normal_sample_id}--{pair_status}.maf"
```

Comment:
I think this might be missing the `sample` in the tumour_id wildcard?
Here is the part from config:
```
# Available wildcards: {unix_group} {seq_type} {tumour_sample_id} {normal_sample_id} {pair_status} {genome_build}
```